conanfile.py

```
@@ -0,0 +1,98 @@
+    homepage = "https://4j-company.github.io/subprojects/mr-math/"
+    url = "https://github.com/4J-company/mr-math"
+
+    topics = ("math", "linear algebra", "computer graphics", "SIMD")
+
+    settings = ("os", "compiler", "build_type", "arch")
+
+    options = {
+        "target": ["library", "tests", "benchmark", "all"],
+        "profiling": [True, False],
+        "extra_optimized": [True, False],
+    }
```

Ruff (0.11.9)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)